.github/workflows/test-all.yml

```diff
@@ -0,0 +1,100 @@
+name: Movapp run all tests
+
+on: [pull_request]
+
+jobs:
+  install_dependencies:
+    runs-on: ubuntu-latest
+    outputs:
+      cache-hit: ${{ steps.cache.outputs.cache-hit }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Cache node modules
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('**/package-lock.json') }}
+
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: npm ci
+      
+  lint:
```

Comment:
z praxe doporučuji taky toto lint a unit_test udělat paralelně.

Ať víš co všechno budeš muset řešit a nezjistíš to až to zase "pošleš" :)

Reply:
Done